unison-src/transcripts/merge.md

```diff
@@ -89,13 +89,13 @@ and `quux` namespaces.
 These test that things we expect to be deleted are still deleted.
 
 ```ucm:error
-.P0> view foo.w
+.> view P0.foo.w
 ```
 
 ```ucm:error
-.P0> view baz.x
+.> view P0.baz.x
 ```
 
 ```ucm:error
-.P0> view quux.x
+.> view P0.quux.x
```

Comment:
Why?

Reply:
To make the suffixes unique and refer to the P0 namespace only. (There’s a foo.w in different namespace at this point in transcript). Mentioned in the writeup but might not have been clear.

Reply:
Sorry, was in a daze and forgot to read it.

Reply:
I have substantially read it now :)